rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeRequest.groovy

```diff
@@ -64,6 +64,8 @@ class BakeRequest {
   String template_file_name
   @ApiModelProperty("A map of key/value pairs to add to the packer command")
   Map extended_attributes
+  @ApiModelProperty("The name of a json file containing key/value pairs to add to the packer command")
+  String var_file_name
```

Comment:
The name of the file will be listed, but not the key/value pairs themselves. They are never populated into the context since they are only deserialized by packer itself.